src/app/public/modules/file-attachment/file-attachment.component.scss

```diff
@@ -0,0 +1,57 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+.sky-file-drop-required {
+  box-shadow: 0 0 8px rgba(239, 64, 68, 0.6);
+  border: 1px solid #ef4044;
+}
+
+.sky-file-input-hidden {
+  display: none;
```

Comment:
We shouldn't have to declare a new class just for hiding an element from the user. Should we use the `hidden` attribute instead?